### ISSUE TITLE
Nothing Happens on Pressing CTRL-P

### ISSUE THREAD

**Issue by Waffelmeister** [thirdy/trademacro#1]
as the title says, nothing shows up

**Comment by thirdy** [comment]
I downloaded from release and tested it to make sure. Works for me.

hover on your item, then ctrl+c, then paste it here?

**Comment by kyri0** [comment]
If this works as other ahk scripts, then you need to have PoE windowed. Full screen won't work. If that's the case, @thirdy , may be for the best for you to notice that on the instructions.

**Comment by Penetros** [comment]
Tried the old pwx pricechecker as I had copy and it worked fine for me, however for some reason longan just won't. Tried running in admin mode and not. Windowed borderless is my default choice. I've given it over 30seconds without moving mouse just in case as well. The jar and ahk file are in the same folder and Java is up to date.

**Comment by thirdy** [comment]
Shouldn't be a problem with PoE if ahk works in general (e.g. pete's price macro or item-info).
longan.ahk uses the same mechanism.

I should note that I did not copy all the 'extra' things that an ahk script should have. Those 'extra' things is probably not extra (or one of those), but actually important to prevent these problems.

Having said that, I'll update the script copying pete's script closely.

---

Another note is that I've found out that it might be possible to do everything in ahk script, (been googling), no need for the java program (which is code I copied from Durian).
